localgov_base.theme

```diff
@@ -65,6 +79,46 @@ function localgov_base_preprocess_page(&$variables) {
   if (!isset($variables['localgov_base_remove_css'])) {
     $variables['#attached']['library'][] = 'localgov_base/global';
   }
+
+  // We have a theme setting boolean for removing the pink background from
+  // unpublished content. If that setting is false,
+  // we will add the 'unpublished' library to our page.
+  dpm(theme_get_setting('localgov_base_add_unpublished_background_colour'));
+  if (theme_get_setting('localgov_base_add_unpublished_background_colour') == TRUE) {
```

Comment:
The '== TRUE' bit of this condition is redundant here (as well as in a few other places in this file).